src/App.js

```diff
@@ -230,209 +258,199 @@ class App extends Component {
-                    title="Github Link"
-                  >
-                    {" "}
-                    <GitMark />{" "}
-                  </a>
-                </div>
-              </div>
-              <div className="movies" id="movie-div">
-                <MovieList
-                  movies={this.state.movies}
-                  setDescription={this.setDescription.bind(this)}
-                />
-                {this.state.isLoading && !this.state.error && (
-                  <div className="skeleton-movies">
-                    <SkeletonLoader />
-                    <SkeletonLoader />
-                    <SkeletonLoader />
-                    <SkeletonLoader />
-                    <SkeletonLoader />
-                  </div>
-                )}
-                {this.state.error && (
-                  <div className="error">
-                    <p className="error-text">
-                      {this.state.searchError !== ""
-                        ? this.state.searchError
-                        : "Oops..An Unknown Error Occured"}{" "}
-                    </p>
-                    {this.state.searchError ? null : (
-                      <button
-                        className="error-retry-btn"
-                        onClick={this.tryAgain.bind(this)}
-                      >
-                        <RetryIcon />
-                        Try Again
-                      </button>
+            <>
+                <GlobalStyles />
+                <div className="App">
+                    <div className="header">
+                    <div className="header-left">
+                        <p> G<span class="em">o</span>phie </p>
+                    </div>
+                    <div className="header-center">
+                        <input
+                        type="text"
+                        ref={this.searchInput}
+                        className="form-control"
+                        placeholder="Search for a movie..."
+                        autoFocus={true}
+                        onKeyPress={this.checkKey.bind(this)}
+                        onChange={this.handleSearchChange.bind(this)}
+                        />
+                    </div>
+                    <div className="header-right">
+                        <button onClick={this.newSearch.bind(this)} className="search-btn" data-tour="my-third-step">
+                            <SearchIcon />
+                        </button>
+                    </div>
+                    </div>
+                    <div className='header-center'>
+                    <input
+                        type="text"
+                        ref={this.searchInput}
+                        className="form-controlb"
+                        placeholder="Search by Year 2020-2018"
+                        autoFocus={true}
+                        onKeyPress={this.setYear}
+                        // onKeyDown ={this.performList}
+                        />
```

Comment:
Make this a **select** tag instead of an **input**

Reply:
Alright

Reply:
good